db/seeds/development/1_people.rb

```diff
@@ -19,15 +19,13 @@ def amount(role_type)
 
 end
 
-puzzlers = [
-  'Andre Kunz',
-  'Andreas Maierhofer',
-  'Mathis Hofer',
-  'Matthias Viehweger',
-  'Pascal Simon',
-  'Pascal Zumkehr',
-  'Roland Studer'
```

Comment:
Einerseits verstehe ich hier die Motivation, Roland ist ja kein Puzzler mehr per se und auch dem Projekt nicht beteiligt. Andererseits is er noch der gemeinsame Nenner, der in jedem Projekt vorhanden ist und daher auch im Cryptopus hinterlegt ist. Wollen wir einen anderen allgemeinen User anlegen? Wenn nicht, dann hinterlege ich die Credentials des root-users...

Reply:
Ich bin sehr dafür, einen generischen User zu verwenden. Selber verwende ich auch fast immer den Root-User, aber der hat halt manchmal bestimmte Quirks, wie den einen den ich [hier](https://github.com/hitobito/hitobito/commit/a4287768100248e22fba29ad3c068ed8f4fb24d3) gefixt habe (weil `can? :show, nil` beim Root User erlaubt ist). Wir hatten mal am Weekly gesagt dass diese Quirks es generell nicht wert sind systematisch gefixt zu werden, das müsste man vielleicht nochmals überdenken wenn wir uns für den Root User entscheiden.